src/lsm/table_iterator.zig

```diff
@@ -9,6 +9,7 @@ const RingBuffer = @import("../ring_buffer.zig").RingBuffer;
 const ManifestType = @import("manifest.zig").ManifestType;
 const GridType = @import("grid.zig").GridType;
 
+/// A TableIterator iterates a table's values in ascending-key order.
```

Comment:
We need to improve this soon to support `direction` so that we can use `TableIterator` for ascending/descending range queries.